components/molecules/AuthSection/auth-section.tsx

```diff
@@ -117,7 +117,16 @@ const AuthSection: React.FC = ({}) => {
                   <Divider type="vertical" className="!h-6 !bg-light-slate-6"></Divider>
                 </>
               )}
-              <button className="relative cursor-pointer" onClick={() => router.push(`/user/notifications`)}>
+              <button
+                key="notifications"
+                className="relative cursor-pointer"
+                onClick={() => {
+                  if (userInfo && userInfo.notification_count > 0) {
+                    setUserInfo({ ...userInfo, notification_count: 0 });
```

Comment:
Does this still work if the page is refreshed? I imagine the user in the database needs to update the notification count?

Reply:
I will give this a thought.

Reply:
It appears to be updating when refreshed. 


https://github.com/open-sauced/app/assets/106407013/4ce1ce7b-3b79-4fcb-a12a-5ca0776f6c1d

Reply:
One last test for the fix. If you log out, close the browser then reopen everything and log in, does it still stay cleared?

Reply:
Yes, it does.


https://github.com/open-sauced/app/assets/106407013/7f7bd57e-e1df-4f84-8b32-7b5ab62fb507